webpack.config.js

```diff
@@ -18,6 +18,8 @@ module.exports = {
   output: {
     filename: 'main.js',
     path: path.resolve(__dirname, 'dist'),
+    clean: true,
+    publicPath: '/TODOLIST/',
```

Comment:
- [ ] I do see the config for the **`dist`** folder but I can't run the project see the screenshot 👇
![ghgh](https://user-images.githubusercontent.com/61562626/129880516-dfab0a89-e383-4d54-a968-d2cfeb300c9d.png)

Reply:
Hey did you run npm install, npm build before running npm start

Reply:
when you set up locally you need to run npm install to install dependencies, then run npm run build, then you can run npm start. Please confirm that you have tried this on your end because it works perfectly on my end

Reply:
Yes, of course or how I can run the project if I don't.

Reply:
it is working in my partner's machine too. besides, you can check it via the live demo link